README.md

```diff
@@ -9,38 +9,42 @@ SAML: https://wiki.oasis-open.org/security/FrontPage
 ECP: http://docs.oasis-open.org/security/saml/Post2.0/saml-ecp/v2.0/saml-ecp-v2.0.html
 
 ## Setup
-To build the project, execute `gradle build` at the project root.
-The `distribution/command-line` module will contain a full package of the deployment after the build.
-
-Tests can be run with the `samlconf` script under `<PATH_TO_PROJECT>/distribution/command-line/build/distributions/samlconf-<VERSION>/bin/`.
-
-The `samlconf` script takes multiple parameters:
+To build the project:
 
-```
-NAME
-       samlconf - Runs the SAML Conformance Tests against an IdP or an SP
+    gradle build
 
-SYNOPSIS
-       samlconf [-i path] [--idpMetadata path]
-           [-p path] [--plugins path] 
-
-DESCRIPTION
-       Runs the SAML Conformance Tests which tests the compliance of an IdP and/or an SP
-       with the SAML Specifications. If a compliance issue is identified, a 
-       SAMLConformanceException will be thrown with an explanation of the error and a direct
-       quote from the specification. All of the parameters are optional and if they are 
-       not provided, the default values will use DDF's parameters. All parameters must 
-       be given one time.
+The `distribution/command-line` module will contain a full package of the deployment after the build.
 
-OPTIONS
-       -i | --idpMetadata path
-           The path to the IdP metadata. If it is not given, the default IdP metadata
-           is /conf/idp-metadata.xml.
-           
-       -p | --plugins path
-            The path to the custom, server-specific plugin implementations. If it is not given, 
-            the default plugin directory is /plugins.
-```
+### Running Test Script
+Upon a successful build, tests can be run with the `samlconf` script found in:
+    
+    distribution/command-line/build/distributions/samlconf-<VERSION>/bin/
```

Comment:
❗️ If the user ran `gradle build installDist` then they'd have an exploded installation in `distribution/command-line/build/install/samlconf` which would contain the script at `distribution/command-line/build/install/samlconf/bin/samlconf`. If they did not run the `installDir` task, they would only have a zip and tar file they could then explode.